frontend/src/Components/LoginElement.css

```diff
@@ -0,0 +1,25 @@
+/*.loginComponentsMaster{*/
```

Comment:
I solved this via html because it looks way better than with my css Skills.
Maybe next Week, when I got more time to test it out, I will change it back to css.